test/language/statements/class/fields-computed-name-propname-constructor.js

```diff
@@ -0,0 +1,34 @@
+// This file was procedurally generated from the following sources:
+// - src/class-fields/propname-constructor.case
+// - src/class-fields/propname-error/cls-decl-computed-name.template
+/*---
+description: Static class fields cannot have PropName 'prototype' (computed)
+esid: sec-class-definitions-static-semantics-early-errors
+features: [class-fields]
+flags: [generated]
+info: |
+    Static Semantics: PropName
+    ...
+    ComputedPropertyName : [ AssignmentExpression ]
+      Return empty.
+
+    // This test file tests the following early error:
+    ClassElement : FieldDefinition;
+      It is a Syntax Error if PropName of FieldDefinition is "constructor".
+
+    FieldDefinition : ClassElementNameInitializer
+      Return PropName of ClassElementName.
+
+    ClassElementName : PropertyName
+      Return PropName of PropertyName.
+
+    ClassElementName : PrivateName
+      Return empty.
+
```

Comment:
Looks like the `negative:` frontmatter is gone? I'm not sure why that would happen, but @leobalter probably knows

Reply:
@rwaldron @leobalter this file should not early error -- so it was not included in the template :)